domain/src/main/java/org/oppia/domain/profile/ProfileTestHelper.kt

```diff
@@ -35,8 +38,9 @@ class ProfileTestHelper @Inject constructor(
         (x + 65).toChar().toString(),
         "123",
         null,
-        allowDownloadAccess = false,
-        isAdmin = false
+        false,
+        R.color.abc_tint_default,
+        false
       )
     }
   }
```

Comment:
Update the comment of `loginToUser`

`/** Login to user profile. */`

Reply:
Updated.